### DISCUSSION TITLE
Possible to configure RGB & 2 separate White channels?

### DISCUSSION THREAD

**Discussion by matt-sullivan** [arendst/Tasmota#12801] [Q&A] [answered]
Is it possible to configure tasmota to use/expose an RGB light and two separate white lights?

I have a H801 wired to an RGB led strip and two separate white lights. I'd like to control each of the 3 physical lights separately. I'm currently using ChannelRemap=128 to expose two light objects, but the two white channels are still conjoined into one logical tasmota light. I could use MultiPwm to create 5 separate channels and I am using home assistant so I could use a template to merge back into one rgb light but it seems more logical to have tasmota treat it as 3 lights if it's possible in any way.

**Comment by s-hadinger** [comment, marked as answer]
No it's not possible.  You can either split RGB and CWW or have 5 independent channels.
